ComputeVisibleServices/function.json

```diff
@@ -0,0 +1,17 @@
+{
+ "scriptFile": "../lib/index.js",
+ "disabled": false,
+ "bindings": [
+    {
+      "type": "cosmosDBTrigger",
+      "name": "visibleServices",
+      "direction": "in",
+      "leaseCollectionName": "leases",
+      "connectionStringSetting": "COSMOSDB_CONNECTION_STRING",
+      "databaseName": "agid-documentdb-test",
```

Comment:
even if the documentaition says that you can use %ENV_VAR% notation, I've experienced it does not work. I've come to the conclusion that we can safely share the same database name between environments in the same way we do for collection names (it will belong anyway to different cosmosdb account with different names). note that collection names are already hardcoded in function.json(s).

Reply:
but how can we share the same database name now? we have already created the databases with different names!

Reply:
renaming them ? =)